ResearchStack2UI/ResearchStack2UI/iOS/RSDStepViewController.swift

```diff
@@ -228,6 +228,21 @@ open class RSDStepViewController : UIViewController, RSDStepViewControllerProtoc
     /// A footer view that includes navigation elements.
     @IBOutlet open var navigationFooter: RSDNavigationFooterView?
     
+    /// The label for displaying step detail text.
+    open var stepTitleLabel: UILabel? {
+        return self.navigationHeader?.titleLabel ?? self.navigationFooter?.titleLabel
+    }
+    
+    /// The label for displaying step detail text.
```

Comment:
not sure what this should say, but prolly not the same as the doc comment for stepDetailLabel...